packages/builder/src/setupTests.ts

```diff
@@ -4,6 +4,8 @@
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
 import "@testing-library/jest-dom/extend-expect";
-import { TextEncoder } from "util";
+import { TextEncoder, TextDecoder } from "util";
 
 global.TextEncoder = TextEncoder;
+// @ts-expect-error We love Jest
```

Comment:
:rofl:

Reply:
One does not simply _use_ Jest. You must configure it. Understand it. Master it. Recite the source code from memory. Offer sacrifices to keep it running.